README.md

```diff
@@ -4,15 +4,15 @@
 
 ## Install
 
-```console
-$ npm install eslint-loader
+```bash
```

Comment:
This is not correct. bash is for bash script, and this is a console snippet. Please just add --save-dev, but not even sure it's mandatory...